app/assets/javascripts/uploadcare/widget/tabs/camera-tab.coffee

```diff
@@ -198,3 +200,26 @@ uploadcare.namespace 'widget.tabs', (ns) ->
 
       @__recorder.stop()
       @__chunks = []
+
+    __guessExtensionByMime: (mime) ->
+      known_containers = {
+        'mp4': 'mp4',
+        'ogg': 'ogg',
+        'webm': 'webm',
+        'quicktime': 'mov',
+        'x-matroska': 'mkv',
```

Comment:
There are all popular video containers. A container (not codecs) defines actual file extension.